core/src/main/java/org/hisp/dhis/android/core/analytics/trackerlinelist/internal/evaluator/NotSupportedEvaluator.kt

```diff
@@ -31,7 +31,7 @@ package org.hisp.dhis.android.core.analytics.trackerlinelist.internal.evaluator
 internal class NotSupportedEvaluator : TrackerLineListEvaluator() {
 
     override fun getCommonSelectSQL(): String {
-        return "Not supported"
+        return "'Not supported'"
```

Comment:
why do we need the comas?

Reply:
The return value is directly used to build a SQL query. If it is not wrapped with commas, it is considered as a column or an operator. It is necessary to wrap it with commas to be considered a string